Makefile

```diff
@@ -12,14 +12,20 @@ install:
 
 ISORT_PARAMS = --ignore-whitespace --settings-path ./ --recursive raiden_contracts/
 
+BLACK_PARAMS = --line-length 99 raiden_contracts/
+
 lint:
+	black --check $(BLACK_PARAMS)
 	flake8 raiden_contracts/
 	pylint --rcfile .pylint.rc --load-plugins pylint_quotes raiden_contracts/
 	isort $(ISORT_PARAMS) --check-only
 
 isort:
 	isort $(ISORT_PARAMS)
 
+black:
```

Comment:
In the services we have an additional `format` target which runs `isort` and `black`, which is quite convenient.

Reply:
Good idea.  Added in this PR.